<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>-7</x>
     <y>1070</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>263</width>
     <height>426</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>时间选择</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dateTimeEdit1">
      <property name="minimumSize">
       <size>
        <width>251</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>251</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>268</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>261</width>
        <height>81</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>261</width>
        <height>81</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{
	border: 2px solid black;

}</string>
      </property>
      <property name="title">
       <string>播放控制</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="leftspeed_btn">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>15</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#leftspeed_btn{
	
	background-image: url(:/Resources/Rsouces/left.png);
	border: 0px;

}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pause_btn">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>15</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#pause_btn{
border:none;background-image:url(:/Resources/Rsouces/pause.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rightspeed_btn">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>15</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#rightspeed_btn{
background-image: url(:/Resources/Rsouces/right.jpg);
border:0;

}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="camera_btn">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>15</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#camera_btn{
background-image: url(:/Resources/Rsouces/camera.png);
border:0;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clip_btn">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>15</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#clip_btn{
background-image: url(:/Resources/Rsouces/clip.png);
border:0;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
